### PR TITLE
WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2 version 12.7.2

### PR DESCRIPTION
**Pull request by Crazy-Clown**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38499)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Looks like this should pass validation, great job!

However, I'm marking as Needs-Author-Feedback just so the mods see this.
Both of us submitted PR's at the same time, and it looks like #38500 has a bit more metadata added to the locale file.

Edit: Whoops, found something that should be updated.

File: manifests/w/WatchGuardTechnologies,Inc/WatchGuardSystemManager12/7/2/12.7.2/WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2
```
Comment (Trenly) [inline manifests/w/WatchGuardTechnologies,Inc/WatchGuardSystemManager12/7/2/12.7.2/WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2.installer.yaml:4]:
```suggestion
PackageIdentifier: WatchGuardTechnologies.WatchGuardSystemManager
```

File: manifests/w/WatchGuardTechnologies,Inc/WatchGuardSystemManager12/7/2/12.7.2/WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2
```
Comment (Trenly) [inline manifests/w/WatchGuardTechnologies,Inc/WatchGuardSystemManager12/7/2/12.7.2/WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: WatchGuardTechnologies.WatchGuardSystemManager
```

File: manifests/w/WatchGuardTechnologies,Inc/WatchGuardSystemManager12/7/2/12.7.2/WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2
```
Comment (Trenly) [inline manifests/w/WatchGuardTechnologies,Inc/WatchGuardSystemManager12/7/2/12.7.2/WatchGuardTechnologies,Inc.WatchGuardSystemManager12.7.2.yaml:4]:
```suggestion
PackageIdentifier: WatchGuardTechnologies.WatchGuardSystemManager
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Since there is already an older version of this package, it should use the same PackageIdentifier as the previous version
image

When changing the PackageIdentifier, you'll also have to change the name of the folder to match

**Comment by Crazy-Clown** [comment]
> Since there is already an older version of this package, it should use the same PackageIdentifier as the previous version image
> 
> When changing the PackageIdentifier, you'll also have to change the name of the folder to match

Yea, for some reason within the existing package, the version field is blank for some reason.
unknown

**Comment by Crazy-Clown** [comment]
@Trenly Are you able to assist with the latest comment from @msftbot ? I don't understand what metadata was not compliant.

**Comment by Crazy-Clown** [comment]
> It is always blank, that is the fault of the app creator. They are not setting the version in Control Panel, and so winget can't determine what version the app is because the publisher does not set the DisplayVersion registry key. The only way to really fix this is to contact the publisher and request that they add the version property in their build scripts

Oh so this value has to be implemented?
version

**Comment by Trenly** [comment]
> Oh so this value has to be implemented? version
> 
> I will approach WatchGuard and ask them to implement this. Thanks for the info on that.

Yes, that is exactly the value that would need to be added. I can provide a command that should fix it on your computer as a workaround, but the best way to get it resolved is for publishers to properly set their app information

**Comment by Crazy-Clown** [comment]
> but the best way to get it resolved is for publishers to properly set their app information

I've put in a request for enhancement with the vendor, will hopefully see a change in the future. Thanks.

**Comment by Trenly** [comment]
> @Trenly I've put in a request for enhancement with the vendor, will hopefully see a change in the future. Thanks. Does this info mean the app will not be published unless it's resolved?

No, it is published now; All it means is that you'll still see the issue where you're stuck in the upgrade loop of unknown version

**Pull request closed without merging**